guides/tools/soft-phone.md

```diff
@@ -13,13 +13,18 @@ Before you configure the Zoiper client you have to create SIP domain on [Twilio
 - SIP URI - URI which you will need to use for Zoiper configuration later
 - REQUEST URL - URL to your application which will handle API requests (see [how to configure ngrok](https://github.com/DVELP/cookbook/blob/master/guides/tools/ngrok.md) to get public url from your localhost)
 
+You need to register the SIP domain.
```

Comment:
Is it required step?

Reply:
It was when I tested it. Zopier recognized the Sip when registered. 
But after has the Zopier with the account registered, disabling the Registration looks to work fine so it is not a required step.